setup.cfg

```diff
@@ -29,3 +29,4 @@ where = src
 
 [options.extras_require]
 dev = pytest
+zstd = zstandard<1
```

Comment:
When following semver, anything below one has an unstable API. So this `<1` is quite meaningless in that regard. But given the developers intention to rewrite the plugin in Rust, maybe it is better to keep this? Am I overthinking this?

Reply:
I think I read this was the recommended way to add the dependency, but don’t remember. Let me check again.

Reply:
I guess I got this from https://python-zstandard.readthedocs.io/en/latest/projectinfo.html:
> There will be some backwards incompatible changes before 1.0, probably in the 0.9 release. This may involve renaming the main module from zstd to zstandard and renaming various types and methods. Pin the package version to prevent unwanted breakage when this change occurs!

I recently read [Should You Use Upper Bound Version Constraints?](https://iscinumpy.dev/post/bound-version-constraints/) (short answer: no), but then maybe this case is different ...

I’m inclined to leave it as it is.

Reply:
Thank you for the extra information and the pointers to the good reads. In this case it is quite likely that libraries/applications that use xopen to manage their compression will not depend on other compression libraries. Also python-zstandard has no dependencies itself. That should limit further annoyances. (Well, it has CFFI as a dependency on PyPy, but that should not cause that much problems.) I think we are good to go.